pollen/src/pollen.pest

```diff
@@ -118,7 +118,10 @@ func_call = { identifier ~ call_args }
 term = _{ literal | obj_initialize | func_call | identifier 
           | "(" ~ expr ~ ")" | "[" ~ expr ~ "]" }
 
-expr = { prefix* ~ term ~ (binop ~ prefix* ~ term ) * }
+array_access = { "[" ~ expr ~ "]" }
+postfix = _{ array_access }
+
+expr = { prefix* ~ term ~ postfix* ~ (binop ~ prefix* ~ term ~ postfix*) * }
```

Comment:
This is beyond the scope of this PR, but I could imagine making a few intermediate nonterminals here to make this more intelligible. Something like this:

```
term_ext = { prefix* ~ term ~ postfix* }
binop_chain = { term_ext ~ (binop ~ term_ext)* }
```

Reply:
Looks great! I went ahead and changed that as part of this PR